### DISCUSSION TITLE
Transitions not working in nested sub-routes

### DISCUSSION THREAD

**Discussion by MattFromGer** [vuejs/router#1977] [Help and Questions]
I want to add a `<transition>` component to a nested sub `<router-view>`, but the animation is not triggering. Furthermore the routing seems buggy when adding the `<transition>` element between `<router-view>` and `<component>`. The sub routing works as expected without the `<transition>` element.

[Stackblitz Example](https://stackblitz.com/edit/vue-router-transition-bug?file=src%2Fcomponents%2FCwithChildren.vue&terminal=dev)

**Comment by doutatsu** [comment]
SImilar question, but I'd like to have different transition for nested routes, but instead I get parent transition applying, not the nested route one
